app/views/lookup_keys/_value.html.erb

```diff
@@ -1,13 +1,42 @@
-<div class="fields">
-  <%= text_f f, :match, :size => "col-md-8",
-    :help_block => popover(_("Explain matchers"), _("Matcher is a combination of an attribute and its value, if they match, the value below would be provided.<br> You may use any attribute foreman knows about, such as facts etc for example: <code> domain = example.com </code> or <code> is_virtual = true</code>"),
-                           :title => _("Matcher"))
-  %>
-  <%= checkbox_f(f, :use_puppet_default, :label => _('Use Puppet default'), :size => "col-md-8",
-                    :help_block => use_puppet_default_help,
-                    :onchange=>'toggleUsePuppetDefaultValue(this, "value")') if is_param %>
-  <%= textarea_f f, :value, :rows => :auto, :value => f.object.value_before_type_cast, :size => "col-md-8",
-                    :disabled => f.object.use_puppet_default,
-                    :fullscreen => :true,
-                    :help_inline => remove_child_link(icon_text("remove"), f, {:title => _('remove value')}) %>
-</div>
+<% if is_template %>
+  <table>
+<% end %>
+    <tr class="fields matchers form-group <%= 'has-error' if f.object.errors.any? %>">
+      <td>
+        <%= f.hidden_field :match, :class => 'match' %>
+        <%= select_tag '', nil, :class => 'matcher_key' %>
```

Comment:
I'll add it but it's problematic in a table, the only way I can think of adding it is with a max-width in the css and even then you would only see `ho..` selected because the `x` takes up space and there is not enough room for more.
This should be a pretty short list and I don't think it's worth adding it but I'll push the changes and maybe someone will have a suggestion on how to make more space in the matcher row.